frontend/packages/operator-lifecycle-manager/src/components/operand/index.tsx

```diff
@@ -437,17 +570,49 @@ export const ProvidedAPIPage = connectToModel((props: ProvidedAPIPageProps) => {
       />
     );
   }
-
+  const providedAPIs = providedAPIsForCSV(csv);
+  const firehoseResourcesForAllNs = firehoseResources(models, providedAPIs, true);
+  const firehoseResourcesForCurrentNs = [
+    { kind: `${kindObj.apiGroup}~${kindObj.apiVersion}~${kindObj.kind}`, namespaced: true },
```

Comment:
Doesn't `namespaced` depend on the kind?

```suggestion
    { kind: `${kindObj.apiGroup}~${kindObj.apiVersion}~${kindObj.kind}`, namespaced: kindObj.namespaced },
```